.changelog/533.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+Use Consul Catalog API instead of Agent API to register APIGateway to prevent GRPC rate limits
+```
```

Comment:
```suggestion
```release-note:bug
Use Consul Catalog API instead of Agent API to register gateways in agentless deployments, to avoid gRPC rate limiting for subsequent API calls handled by Consul servers other than the one with which the gateway was registered 
```
```